numpy/core/arrayprint.py

```diff
@@ -701,40 +707,32 @@ def __init__(self, data, precision, floatmode, suppress_small, sign=False, **kwa
         self.fillFormat(data)
 
     def fillFormat(self, data):
-        with errstate(all='ignore'):
```

Comment:
Why did this line go away?

Reply:
Oh I think I need to keep it. It must be because the division operation can overflow, eg `1.0/np.float64(2.2250738585072e-310)`.

Reply:
Oops, I just made the other fixes online.

Reply:
You may need to update from origin.

Reply:
Maybe push it to a deeper scope this time